src/web/src/components/Posts/Commits.tsx

```diff
@@ -61,7 +66,7 @@ const Commits = ({ commitUrls }: Props) => {
               href={`https://github.com${url}`}
               rel="bookmark"
               target="_blank"
-              title={'Commit ' + getCommitNumber(url)}
+              title={`${getCommitInfo(url)} Commit Hash ${getCommitNumber(url)}`}
```

Comment:
Let's drop the `Hash `, the rest is great.

Reply:
Ready to merge!